### PR TITLE
IR-468: blocked-edges/4.14.*: Declare AzureRegistryImageMigrationUserProvisioned

### PR DESCRIPTION
**Pull request by wking**
Generated by writing the 4.14.15 risk, and then copying out to later 4.14.z through the most recent 4.14.21:

```console
$ tail -n2  channels/candidate-4.14.yaml
- 4.14.20
- 4.14.21
```

with:

```console
$ for Z in $(seq 16 21); do sed "s/4.14.15/4.14.${Z}/" blocked-edges/4.14.15-AzureRegistryImageMigrationUserProvisioned.yaml > "blocked-edges/4.14.${Z}-AzureRegistryImageMigrationUserProvisioned.yaml"; done
```

I'm not currently clear on whether updates from `4.14.(<15)` are also exposed; if they are, we can follow up with a bump to the `from` regular expression.

### CODE REVIEW

**Comment by wking** [comment]
Checking the PromQL vs. a GCP cluster I have access to, and it successfully evaluates as 0 (risk does not match this cluster):

image

**Review by petr-muller — APPROVED** [review]
LGTM, not sure if you want to merge with https://issues.redhat.com/browse/IR-468 still incomplete => holding. Unhold at will.

/hold

**Review by petr-muller — APPROVED** [review]

**Comment by petr-muller** [comment]
/lgtm

Poor Prow :D

**Comment by wking** [comment]
Expanded to cover 4.14.old -> 4.14.(>=15), and copied Flavian's impact statement up into the IR-468 `Description`.

/hold cancel

**Pull request merged**
